src/lib/Scenes/Collection/Components/CollectionHubsRails/ArtistSeries/__tests__/CollectionArtistSeriesRail-tests.tsx

```
@@ -10,246 +11,438 @@ import {
+
+describe("Artist Series Rail", () => {
+  const trackEvent = jest.fn()
+  const getWrapper = async () => {
+    return renderRelayTree({
+      Component: (props: any) => (
+        <Theme>
+          <CollectionArtistSeriesRailContainer
+            collection={props.marketingCollection}
+            collectionGroup={props.marketingCollection.linkedCollections[0]}
+            {...props}
+          />
+        </Theme>
+      ),
+      query: graphql`
+        query CollectionArtistSeriesRailTestsQuery @raw_response_type {
+          marketingCollection(slug: "photography") {
+            ...CollectionArtistSeriesRail_collection
+            linkedCollections {
+              groupType
+              ...CollectionArtistSeriesRail_collectionGroup
+            }
           }
         }
-      }
-    `,
-    mockData: { marketingCollections: CollectionHubRailsArtistSeriesFixture },
-  })
-})
+      `,
+      mockData: CollectionHubRailsArtistSeriesFixture,
+    })
+  }
 
-describe("Trending Artists Rail", () => {
-  let props: any /* STRICTNESS_MIGRATION */
   beforeEach(() => {
-    props = {
-      collectionGroup: { ...CollectionHubRailsArtistSeriesFixture },
-    }
+    const mockTracking = useTracking as jest.Mock
+    mockTracking.mockImplementation(() => {
+      return {
+        trackEvent,
+      }
+    })
   })
 
-  it("renders the Trending Artists Series rail component", () => {
-    const wrapper = mount(
-      <Theme>
-        <CollectionArtistSeriesRail {...props} />
-      </Theme>
-    )
-
-    expect(wrapper.find(GenericArtistSeriesRail)).toHaveLength(1)
+  afterEach(() => {
+    jest.clearAllMocks()
   })
 
-  it("renders three artist series in the Trending Artists Series", () => {
-    const wrapper = mount(
-      <Theme>
-        <CollectionArtistSeriesRail {...props} />
-      </Theme>
-    )
-
-    expect(wrapper.find(ArtworkImageContainer)).toHaveLength(3)
+  it("renders without throwing an error", async () => {
```

Heads up @ashleyjelks for these kinds of tests, we need to both render the component + assert _something_ (otherwise it'll silently fail!).